src/System.Diagnostics.EventLog/ref/System.Diagnostics.EventLog.cs

```diff
@@ -154,3 +157,374 @@ public enum OverflowAction
         OverwriteOlder = 1,
     }
 }
+namespace System.Diagnostics.Eventing.Reader
```

Comment:
@vancem I assume you don't have any particular position on this.
@weshaggard do you have a SHA handy illustrating everything necessary to add a new library?

Reply:
>@weshaggard do you have a SHA handy illustrating everything necessary to add a new library?

I don't have one handy but I suggest to just follow the pattern of the EventLog library.

Reply:
Answering @weshaggard  question: Why place EventLogReader into System.Diagnostics.EventLog rather than making a new System.Diagnostics.EventLog.Reader dll?

Answer: complexity has to earn its keep.    It is clearly more complex to split the functionality into two DLLs (more files, more projects and clients need to know more to find the right DLL (sometimes)...).   The value of splitting is actually unclear at this level of granularity (splitting a 10MB DLL is likely to be good, splitting a 50K DLL is probably bad).   

This is especially true with things like EventLogReader, which are not likely to be used widely or change much going forward.    Simplicity should rule.   

I am happy to weigh the merits of two DLLs, but we should be clear on the benefits before we do.   The default should be to keep it simple (which in this case is one DLL).

Reply:
@vancem If adding this functionality here makes sense and doesn't cause any unnecessary increase in size and dependencies then I agree adding it is probably OK. Correct me if I'm wrong but aren't these 2 different technologies for doing EventLog stuff? That is why I had the inclination to split them.